include/kllvm/codegen/DecisionParser.h

```diff
@@ -5,8 +5,8 @@
 
 namespace kllvm {
 
-DecisionNode *parseYamlDecisionTreeFromString(std::string yaml, int numSubjects, const llvm::StringMap<KOREObjectSymbol *> &syms, const llvm::StringMap<KOREObjectCompositeSort *> &sorts);
-DecisionNode *parseYamlDecisionTree(std::string filename, int numSubjects, const llvm::StringMap<KOREObjectSymbol *> &syms, const llvm::StringMap<KOREObjectCompositeSort *> &sorts);
+DecisionNode *parseYamlDecisionTreeFromString(std::string yaml, const llvm::StringMap<KOREObjectSymbol *> &syms, const llvm::StringMap<KOREObjectCompositeSort *> &sorts);
```

Comment:
Why was `numSubjects` removed? I think we will want to support trees that start with multiple variables available to bind.

Reply:
it is no longer needed; we can now infer everything needed to generate the decision tree from the decision tree itself.